docs/EVG.md

```diff
@@ -345,6 +345,12 @@ Each CA MUST publicly disclose its Certificate Policy and/or Certification Pract
 
 Effective as of 31 May 2018, the CA's Certificate Policy and/or Certification Practice Statement MUST be structured in accordance with RFC 3647. Prior to 31 May 2018, the CA's Certificate Policy and/or Certification Practice Statement MUST be structured in accordance with either RFC 2527 or RFC 3647. The Certificate Policy and/or Certification Practice Statement MUST include all material required by RFC 3647 or, if structured as such, RFC 2527.
 
+Effective as of 1 September 2020, the CA MUST publicly disclose through an appropriate and readily accessible online means that is available on a 24x7 basis every Incorporating Agency and Registration Agency it has determined satisfies the requirements in these Guidelines. The CA SHALL document where to obtain this information within Section 3.2 of the CA's Certificate Policy and/or Certification Practice Statement. This Agency Information SHALL include the following:
+* Sufficient information to unambiguously identify the Incorporating Agency or Registration Agency (such as a name, jurisdiction, and website); and,
+* The accepted values for the `subject:jurisdictionLocalityName` (OID: 1.3.6.1.4.1.311.60.2.1.1), `subject:jurisdictionStateOrProvinceName` (OID: 1.3.6.1.4.1.311.60.2.1.2), and `subject:jursidictionCountryName` (OID: 1.3.6.1.4.1.311.60.2.1.3) fields when a certificate is issued using information from that Incorporating Agency or Registration Agency, indicating the jurisidction(s) that the Agency is appropriate for; and,
+* The accepted or allowed form or syntax of the Registration Number used by the Incorporating Agency or Registration Agency, if known; and,
+* A revision history that includes a unique version number and date of publication for any additions, modifications, or removals from this list.
```

Comment:
Just want to clarify that it is acceptable that the revision history is generated by the content management or publication system, instead of being part of the document itself.

For example, we are considering publishing the list to a public github repo, where github would take care of listing the versions and the changes between versions.

Reply:
@timfromdigicert

If your CPS disclosed the GitHub repository, that would seem to suffice, right? There's no inherent requirement that it's a single "document". Did you feel that was not clear?

Reply:
I think it's possible to infer that such an implementation is compliant, but I don't think it's as clear as it could be.  Also, I wanted to make sure no one disagrees that such an implementation is compliant ...

Reply:
Perhaps - "A version history (either within the disclosure, or via a content management system or document versioning system) that includes..."

Reply:
Hrm, the problem with that is "within the disclosure" seems to promote the very idea this was intentionally trying to be permissive on: that it doesn't have to be a single document. I was trying to aim for maximum flexibility, which does mean not stating things explicitly, especially when they read as constraining (e.g. your language there would seem to be "either/or" and only allow those two implementations)

I worry it also might end up for the wrong approach, where simply disclosing that "versions exist" would be seen as sufficient, which is why I went with "revision history".

For better or worse, I intentionally left terrible loopholes in precisely to try and make it easy for CAs to digest. Like, read literally, a CA could simply point to their website and argue that it meets the requirements. I'm all for wanting to improve that precision, but I feel the more that's added, the greater risk that's introduced, and I'm just trying to get the bare minimum of effort out of CAs so we can make progress in the WG.

Reply:
Hmm, I actually used disclosure instead of document to try to avoid that interpretation.  If we're trying to be permissive, there could an "or elsewhere" at the end.  The "versions exist" loophole did occur to me, as did:

2020-05-19, version 2020.05.19 - updated with latest acceptable sources
2020-05-18, version 2020.05.18 - updated with latest acceptable sources
2020-05-17, version 2020.05.17 - updated with latest acceptable sources
2020-05-16, version 2020.05.16 - updated with latest acceptable sources

But I think attempting to squash these loopholes would just encourage more creative loophole attempts.

Perhaps we just leave the language alone, and refer anyone who questions whether a github version history is acceptable (for example) to this conversation.

Reply:
OK, I'm going to "Resolve" this conversation (you can still link to it), but I suspect we do still need changes because of https://github.com/sleevi/cabforum-docs/pull/11/files#r425350739